tools/devtool

```diff
@@ -660,7 +660,7 @@ cmd_test() {
         -v /boot:/boot \
         ${ramdisk_args} \
         -- \
-        pytest "$@"
+        pytest -W error::pytest.PytestUnraisableExceptionWarning -W error::pytest.PytestUnhandledThreadExceptionWarning "$@"
```

Comment:
did you see this work in your testing?

Reply:
Yep, there's old buildkite runs where it triggered, e.g. https://buildkite.com/firecracker/firecracker-pr/builds/1048